proto/cosmos/group/v1/tx.proto

```diff
@@ -342,8 +347,8 @@ message MsgExec {
   // proposal is the unique ID of the proposal.
   uint64 proposal_id = 1;
 
-  // signer is the account address used to execute the proposal.
-  string signer = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // address is the account address used to execute the proposal.
+  string address = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
```

Comment:
We use `address` a bit everywhere in the group proto files, but `signer` here. Proposing to make it consistent.